### ISSUE TITLE
Cannot install and run in Wine

### ISSUE THREAD

**Issue by vliaskov** [octgn/OCTGN#1583]
Since OCTGN dropped support for Windows XP, I cannot run OCTGN on wine, even though I am trying to use Windows 7 and above. Is dotnet45 an official requirement now?

I am still following this guide (which by the way probably needs to be updated to reflect dotnet45 requirements and WinXP obsolete status):
https://github.com/octgn/OCTGN/wiki/OCTGN-on-non-Windows-systems

Doing "WINEARCH=win32 WINEPREFIX=~/.wine-octgn winecfg", I know choose Windows7 (or above) instead of default WindowsXP.

Next I use latest winetricks to install dotnet45:
WINEPREFIX=~/.wine-octgn winetricks -q vcrun2010 dotnet45
However, dotnet45 installation downgrades the system to Windows2003:
"Setting Windows version to 2003, otherwise applications using .NET 4.5 will subtly fail"

And OCTGN setup naturally complains:
WINEPREFIX=~/.wine-octgn wine ~/OCTGN-Setup-3.1.255.0.exe
"You are currently running Windows XP, which is incompatible with .net 4.5. OCTGN no longer runs on Windows XP. We recommend updating to at least Windows 7."

So I switch back to Windows 7 by rerunning winecfg. OCTGN seems to install (completed message is output in installation GUI", but wine messages on cli are:

fixme:ras:RasEnumConnectionsW RAS support is not implemented! Configure program to use LAN connection/winsock instead!
fixme:winsock:WSAIoctl -> SIO_ADDRESS_LIST_CHANGE request: stub
fixme:winsock:WSAIoctl -> SIO_ADDRESS_LIST_CHANGE request: stub
fixme:ras:RasConnectionNotificationW (0xffffffff,0x470,0x00000003),stub!
fixme:crypt:SystemFunction041 (0x65e0e9c, 10, 0): stub [RtlDecryptMemory]
fixme:winhttp:WinHttpDetectAutoProxyConfigUrl discovery via DHCP not supported
fixme:nls:LCMapStringEx unsupported lparam 17ea48
fixme:nls:LCMapStringEx unsupported lparam 17ea48
fixme:nls:LCMapStringEx unsupported lparam 17ea48

Process is terminated due to StackOverflowException.
err:ntdll:RtlpWaitForCriticalSection section 0x110060 "heap.c: main process heap section" wait timed out in thread 0046, blocked by 0039, retrying (60 sec)
err:ntdll:RtlpWaitForCriticalSection section 0x110060 "heap.c: main process heap section" wait timed out in thread 0044, blocked by 0039, retrying (60 sec)
err:ntdll:RtlpWaitForCriticalSection section 0x110060 "heap.c: main process heap section" wait timed out in thread 0038, blocked by 0039, retrying (60 sec)

Then trying to run the installed OCTGN, I get a crash: 
WINEPREFIX=~/.wine-octgn wine explorer /desktop=octgn,1440x900 ~/OCTGN/OCTGN/OCTGN.exe

fixme:advapi:RegisterEventSourceW ((null),L".NET Runtime"): stub
fixme:advapi:ReportEventW (0xcafe4242,0x0001,0x0000,0x00000402,(nil),0x0001,0x00000000,0x782c8f4,(nil)): stub
err:eventlog:ReportEventW L"Application: OCTGN.exe\nFramework Version: v4.0.30319\nDescription: The process was terminated due to an unhandled exception.\nException Info: System.AccessViolationException\nStack:\n   at System.Net.UnsafeNclNativeMethods+SafeNetHandlesXPOrLater.freeaddrinfo(IntPtr)\n   at System.Net.SafeFreeAddrIn"...
fixme:advapi:DeregisterEventSource (0xcafe4242) stub

Unhandled Exception: System.AccessViolationException: Attempted to read or write protected memory. This is often an indication that other memory is corrupt.
   at System.Net.UnsafeNclNativeMethods.SafeNetHandlesXPOrLater.freeaddrinfo(IntPtr info)
   at System.Net.SafeFreeAddrInfo.ReleaseHandle()
   at System.Runtime.InteropServices.SafeHandle.InternalDispose()
   at System.Runtime.InteropServices.SafeHandle.Dispose(Boolean disposing)
   at System.Net.Dns.TryGetAddrInfo(String name, AddressInfoHints flags, IPHostEntry& hostinfo)
   at System.Net.Dns.TryInternalResolve(String hostName, IPHostEntry& result)
   at System.Net.ServicePoint.GetIPAddressInfoList(Int32& currentIndex, IPAddress[] addresses)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   at System.Net.ServicePoint.GetConnection(PooledStream PooledStream, Object owner, Boolean async, IPAddress& address, Socket& abortSocket, Socket& abortSocket6)
   at System.Net.PooledStream.Activate(Object owningObject, Boolean async, GeneralAsyncDelegate asyncCallback)
   at System.Net.Connection.CompleteStartConnection(Boolean async, HttpWebRequest httpWebRequest)
   at System.Net.Connection.CompleteStartRequest(Boolean onSubmitThread, HttpWebRequest request, TriState needReConnect)
   at System.Net.Connection.SubmitRequest(HttpWebRequest request, Boolean forcedsubmit)
   at System.Net.ServicePoint.SubmitRequest(HttpWebRequest request, String connName)
   at System.Net.HttpWebRequest.SubmitRequest(ServicePoint servicePoint)
   at System.Net.HttpWebRequest.BeginGetRequestStream(AsyncCallback callback, Object state)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncImpl(Func`3 beginMethod, Func`2 endFunction, Action`1 endAction, Object state, TaskCreationOptions creationOptions)
   at System.Net.WebRequest.<GetRequestStreamAsync>b__3()
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
   at System.Threading.Tasks.Task.ExecutionContextCallback(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot)
   at System.Threading.Tasks.Task.ExecuteEntry(Boolean bPreventDoubleExecution)
   at System.Threading.Tasks.Task.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
   at System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()
wine: Unhandled page fault on write access to 0x00c9a11c at address 0x7bc49576 (thread 003f), starting debugger...
Unhandled exception: page fault on write access to 0x00c9a11c in 32-bit code (0x7bc49576).
Register dump:
 CS:0023 SS:002b DS:002b ES:002b FS:0063 GS:006b
 EIP:7bc49576 ESP:0782da2c EBP:0782db28 EFLAGS:00010202(  R- --  I   - - - )
 EAX:00c9a118 EBX:7bcbf000 ECX:065f4fc0 EDX:0202b220
 ESI:0641d000 EDI:065f5028
Stack dump:
0x0782da2c:  00000002 00000002 065f4fd0 065f5028
0x0782da3c:  00000000 00000000 00000000 00000000
0x0782da4c:  11eb3165 00000000 00000000 4d430003
0x0782da5c:  4d430003 00000000 00000000 00000000
0x0782da6c:  00000000 00000000 4d430001 065f4fc0
0x0782da7c:  00000010 00000000 00000000 00000000
000c: sel=0067 base=00000000 limit=00000000 32-bit r-x
Backtrace:
=>0 0x7bc49576 in ntdll (+0x39576) (0x0782db28)
  1 0x7bc4983c in ntdll (+0x3983b) (0x0782dbc8)
  2 0x7bc4a55a RtlFreeHeap+0xc9() in ntdll (0x0782dc38)
  3 0xf65a0842 WS_freeaddrinfo+0x41() in ws2_32 (0x0782dc78)
  4 0x07471077 (0x0782dcb8)
  5 0x074710c3 (0x0782dcc0)
  6 0x00b82652 in clr (+0x2651) (0x0782dccc)
  7 0x00b9264f in clr (+0x1264e) (0x0782dd20)
  8 0x00b99dd2 in clr (+0x19dd1) (0x0782dd58)
  9 0x00baaace in clr (+0x2aacd) (0x0782dda8)
  10 0x00baaa7b in clr (+0x2aa7a) (0x0782ddec)
  11 0x00baa9eb in clr (+0x2a9ea) (0x0782de18)
  12 0x00baab51 in clr (+0x2ab50) (0x0782dea8)
  13 0x03df1e78 (0x0782deb0)
  14 0x07470dfc (0x0782df14)
  15 0x07503280 (0x0782df34)
  16 0x0750307b (0x0782df84)
  17 0x07502cf5 (0x0782dfcc)
  18 0x07502a3f (0x0782e00c)
  19 0x07502868 (0x0782e054)
  20 0x07502711 (0x0782e090)
  21 0x0750251a (0x0782e0bc)
  22 0x07501c16 (0x0782e10c)
  23 0x075006b0 (0x0782e148)
  24 0x074fe214 (0x0782e174)
  25 0x074fa055 (0x0782e1b0)
  26 0x074f99f8 (0x0782e1e4)
  27 0x074f9771 (0x0782e210)
  28 0x074f9671 (0x0782e220)
  29 0x06bd39c6 (0x0782e244)
  30 0x06bd3976 (0x0782e2b0)
  31 0x05565087 (0x0782e2c4)
  32 0x06bd3793 (0x0782e310)
  33 0x06bd32fe (0x0782e320)
  34 0x06bd3178 (0x0782e374)
  35 0x06bd1f76 (0x0782e384)
  36 0x00b9264f in clr (+0x1264e) (0x0782e3d8)
  37 0x00b92e95 in clr (+0x12e94) (0x0782e44c)
  38 0x00c8f1cc in clr (+0x10f1cb) (0x0782e4c4)
  39 0x00bf7d4d in clr (+0x77d4c) (0x0782e4dc)
  40 0x00bf7dbb in clr (+0x77dba) (0x0782e584)
  41 0x00bf7e88 in clr (+0x77e87) (0x0782e5e0)
  42 0x00b8c3bf in clr (+0xc3be) (0x0782e604)
  43 0x00c8f164 in clr (+0x10f163) (0x0782e6b8)
  44 0x00c6f54d in clr (+0xef54c) (0x0782e734)
  45 0x00c9a0cf in clr (+0x11a0ce) (0x0782e9d8)
  46 0x7bc799b0 call_thread_func_wrapper+0xb() in ntdll (0x0782e9e8)
  47 0x7bc7c93d call_thread_func+0x7c() in ntdll (0x0782eab8)
  48 0x7bc7998e RtlRaiseException+0x21() in ntdll (0x0782ead8)
  49 0x7bc82dce in ntdll (+0x72dcd) (0x0782f358)
  50 0xf74f1f70 start_thread+0xcf() in libpthread.so.0 (0x0782f428)
  51 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  52 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  53 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  54 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  55 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  56 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  57 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  58 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  59 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  60 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  61 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  62 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  63 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  64 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  65 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  66 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  67 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  68 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  69 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  70 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  71 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  72 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  73 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  74 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  75 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  76 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  77 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  78 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  79 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  80 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  81 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  82 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  83 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  84 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  85 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  86 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  87 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  88 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  89 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  90 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  91 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  92 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  93 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  94 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  95 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  96 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  97 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  98 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  99 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  100 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  101 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  102 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  103 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  104 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  105 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  106 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  107 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  108 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  109 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  110 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  111 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  112 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  113 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  114 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  115 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  116 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  117 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  118 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  119 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  120 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  121 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  122 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  123 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  124 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  125 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  126 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  127 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  128 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  129 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  130 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  131 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  132 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  133 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  134 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  135 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  136 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  137 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  138 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  139 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  140 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  141 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  142 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  143 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  144 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  145 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  146 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  147 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  148 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  149 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  150 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  151 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  152 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  153 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  154 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  155 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  156 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  157 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  158 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  159 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  160 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  161 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  162 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  163 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  164 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  165 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  166 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  167 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  168 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  169 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  170 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  171 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  172 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  173 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  174 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  175 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  176 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  177 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  178 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  179 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  180 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  181 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  182 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  183 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  184 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  185 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  186 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  187 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  188 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  189 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  190 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  191 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  192 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  193 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  194 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  195 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  196 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  197 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  198 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  199 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
  200 0xf7422bee __clone+0x5d() in libc.so.6 (0x00000000)
0x7bc49576: movl    %edx,0x4(%eax)
Modules:
Module  Address         Debug info  Name (113 modules)
PE    400000-  844000   Deferred        octgn
PE    b80000- 1212000   Export          clr
PE   1220000- 12f3000   Deferred        msvcr110_clr0400
PE   3d60000- 3dce000   Deferred        clrjit
PE   4ae0000- 4af2000   Deferred        nlssorting
PE   4b50000- 4ced000   Deferred        wpfgfx_v0400
PE   5070000- 5130000   Deferred        presentationnative_v0400
PE   7160000- 746b000   Deferred        system.data
PE   8590000- 85d5000   Deferred        system.transactions
PE   ae40000- ae4a000   Deferred        wminet_utils
PE  10000000-1007a000   Deferred        mscoreei
PE  5e0d0000-5e1a4000   Deferred        diasymreader
PE  74e60000-74e79000   Deferred        wmiutils
PE  79000000-7904a000   Deferred        mscoree
ELF 7b800000-7ba5b000   Deferred        kernel32<elf>
  -PE  7b810000-7ba5b000   \               kernel32
ELF 7bc00000-7bcdb000   Dwarf           ntdll<elf>
  -PE  7bc10000-7bcdb000   \               ntdll
ELF 7bf00000-7bf04000   Deferred        <wine-loader>
ELF 7ed4c000-7ed59000   Deferred        libnss_files.so.2
ELF 7ed59000-7ed65000   Deferred        libnss_nis.so.2
ELF 7ed65000-7ed7e000   Deferred        libnsl.so.1
ELF 7ef7e000-7efc4000   Deferred        libm.so.6
ELF f5799000-f57b1000   Deferred        libresolv.so.2
ELF f57b1000-f58c0000   Deferred        opengl32<elf>
  -PE  f57d0000-f58c0000   \               opengl32
ELF f58c0000-f5a00000   Deferred        wined3d<elf>
  -PE  f58d0000-f5a00000   \               wined3d
ELF f5b43000-f5b80000   Deferred        winhttp<elf>
  -PE  f5b50000-f5b80000   \               winhttp
ELF f5b80000-f5b9b000   Deferred        rasapi32<elf>
  -PE  f5b90000-f5b9b000   \               rasapi32
ELF f5b9b000-f5c43000   Deferred        msvcrt<elf>
  -PE  f5bb0000-f5c43000   \               msvcrt
ELF f5c43000-f5d00000   Deferred        windowscodecs<elf>
  -PE  f5c50000-f5d00000   \               windowscodecs
ELF f6007000-f600e000   Deferred        libnss_dns.so.2
ELF f600e000-f6035000   Deferred        dxgi<elf>
  -PE  f6010000-f6035000   \               dxgi
ELF f6035000-f6052000   Deferred        libgcc_s.so.1
ELF f6058000-f607e000   Deferred        iphlpapi<elf>
  -PE  f6060000-f607e000   \               iphlpapi
ELF f607e000-f60b0000   Deferred        wbemprox<elf>
  -PE  f6080000-f60b0000   \               wbemprox
ELF f60b0000-f60c6000   Deferred        dwmapi<elf>
  -PE  f60c0000-f60c6000   \               dwmapi
ELF f60c6000-f60ee000   Deferred        mpr<elf>
  -PE  f60d0000-f60ee000   \               mpr
ELF f60ee000-f616a000   Deferred        wininet<elf>
  -PE  f6100000-f616a000   \               wininet
ELF f616a000-f620c000   Deferred        urlmon<elf>
  -PE  f6180000-f620c000   \               urlmon
ELF f620c000-f6243000   Deferred        uxtheme<elf>
  -PE  f6210000-f6243000   \               uxtheme
ELF f6243000-f634a000   Deferred        comctl32<elf>
  -PE  f6250000-f634a000   \               comctl32
ELF f634a000-f657d000   Deferred        shell32<elf>
  -PE  f6360000-f657d000   \               shell32
ELF f657d000-f65b3000   Dwarf           ws2_32<elf>
  -PE  f6580000-f65b3000   \               ws2_32
ELF f65b3000-f65d8000   Deferred        imm32<elf>
  -PE  f65c0000-f65d8000   \               imm32
ELF f65d8000-f670e000   Deferred        oleaut32<elf>
  -PE  f65f0000-f670e000   \               oleaut32
ELF f670e000-f6732000   Deferred        dwrite<elf>
  -PE  f6710000-f6732000   \               dwrite
ELF f6732000-f6801000   Deferred        crypt32<elf>
  -PE  f6740000-f6801000   \               crypt32
ELF f6801000-f6843000   Deferred        rsaenh<elf>
  -PE  f6810000-f6843000   \               rsaenh
ELF f6843000-f68c4000   Deferred        rpcrt4<elf>
  -PE  f6850000-f68c4000   \               rpcrt4
ELF f68c4000-f6a00000   Deferred        ole32<elf>
  -PE  f68e0000-f6a00000   \               ole32
ELF f6b27000-f6b2d000   Deferred        libxfixes.so.3
ELF f6b2d000-f6b38000   Deferred        libxcursor.so.1
ELF f6b38000-f6b48000   Deferred        libxi.so.6
ELF f6b48000-f6b4c000   Deferred        libxcomposite.so.1
ELF f6b4c000-f6b57000   Deferred        libxrandr.so.2
ELF f6b57000-f6b62000   Deferred        libxrender.so.1
ELF f6b62000-f6b68000   Deferred        libxxf86vm.so.1
ELF f6b68000-f6b6c000   Deferred        libxinerama.so.1
ELF f6b6c000-f6b73000   Deferred        libxdmcp.so.6
ELF f6b73000-f6b77000   Deferred        libxau.so.6
ELF f6b77000-f6b99000   Deferred        libxcb.so.1
ELF f6b99000-f6ccd000   Deferred        libx11.so.6
ELF f6ccd000-f6ce0000   Deferred        libxext.so.6
ELF f6ce0000-f6cf6000   Deferred        bcrypt<elf>
  -PE  f6cf0000-f6cf6000   \               bcrypt
ELF f6d1c000-f6dae000   Deferred        winex11<elf>
  -PE  f6d30000-f6dae000   \               winex11
ELF f6e51000-f6e7a000   Deferred        libexpat.so.1
ELF f6e7a000-f6eb5000   Deferred        libfontconfig.so.1
ELF f6eb5000-f6edd000   Deferred        libpng12.so.0
ELF f6edd000-f6ef7000   Deferred        libz.so.1
ELF f6ef7000-f6f97000   Deferred        libfreetype.so.6
ELF f6fd3000-f70f0000   Deferred        gdi32<elf>
  -PE  f6fe0000-f70f0000   \               gdi32
ELF f70f0000-f724a000   Deferred        user32<elf>
  -PE  f7100000-f724a000   \               user32
ELF f724a000-f72c4000   Deferred        shlwapi<elf>
  -PE  f7260000-f72c4000   \               shlwapi
ELF f72c4000-f7336000   Deferred        advapi32<elf>
  -PE  f72d0000-f7336000   \               advapi32
ELF f7337000-f74e5000   Dwarf           libc.so.6
ELF f74e5000-f74ea000   Deferred        libdl.so.2
ELF f74eb000-f7507000   Dwarf           libpthread.so.0
ELF f7507000-f7510000   Deferred        libnss_compat.so.2
ELF f7528000-f7542000   Deferred        version<elf>
  -PE  f7530000-f7542000   \               version
ELF f7543000-f76f8000   Dwarf           libwine.so.1
ELF f76fa000-f771c000   Deferred        ld-linux.so.2
ELF f771e000-f771f000   Deferred        [vdso].so
Threads:
process  tid      prio (all id:s are in hex)
0000000e services.exe
    00000028    0
    00000027    0
    00000020    0
    00000016    0
    00000010    0
    0000000f    0
00000012 mscorsvw.exe
    0000001d    0
    0000001c    0
    00000019    0
    00000013    0
0000001e winedevice.exe
    00000026    0
    00000023    0
    00000022    0
    0000001f    0
00000024 plugplay.exe
    0000002a    0
    00000029    0
    00000025    0
00000047 explorer.exe
    0000000d    0
    0000000b    0
0000002c (D) Z:\home\vliaskov\OCTGN\OCTGN\OCTGN.exe
    0000003a    0
    0000001b    0
    00000009    0
    0000003f    0 <==
    00000040    0
    00000031    0
    00000032    0
    0000002f    0
    00000030    2
    0000002d    0
    0000002e    0
